cloudferrylib/utils/utils.py

```diff
@@ -507,3 +516,16 @@ def import_class_by_string(name):
     for comp in module[1:]:
         mod = getattr(mod, comp)
     return getattr(mod, class_name)
+
+
+def get_remote_file_size(host, file_path, ssh_user, ssh_sudo_password):
+    """ Return file size in bytes on the remote host.
+
+    :param host: Remote host,
+    :param file_path: Full file path,
+    :return: File size in bytes.
+    """
+
+    with settings(host_string=host, user=ssh_user, password=ssh_sudo_password):
+        size = run('wc -c < %s' % file_path)
```

Comment:
Not sure if it's safe to retrieve file size with `wc`:
 - it might not work correctly for binary files
 - in this particular case it will count the number of *bytes* in file, which is not always equal to the file size.

I would recommend using `stat` instead.